src/main/java/com/revature/model/Job.java

```diff
@@ -24,46 +24,40 @@
  */
 @Entity
 @Table(name = "jobs")
+
 public class Job {
-	
 	@Id
 	@Column(name = "job_id")
 	@GeneratedValue(strategy = GenerationType.IDENTITY)
 	/** An integer that uniquely identifies this job. */
 	private int id;
-
 	@Column(name = "job_title")
 	/** The name of this job. */
 	private String title;
-
 	@Column(name = "job_description", length=10_000)
 	/** A description of this job, with a maximum length of 10,000 characters. */
 	private String description;
-	
 	@Column(name = "job_location")
 	/** A string identifying where the job takes place. */
 	private String location;
-
 	@ManyToMany(fetch = FetchType.EAGER, cascade = CascadeType.ALL)
 	@JoinTable(name = "job_skills", joinColumns = { @JoinColumn(name = "job_id") }, inverseJoinColumns = {
 			@JoinColumn(name = "skill_id") })
 	/** A set of skills that candidates applying to this job are expected to have. */
 	private Set<Skill> skills = new HashSet<>();
-
 	@Column(name = "job_isFilled")
 	/** Returns true if the job opening is currently filled, and false otherwise. */
 	private boolean isFilled;
-	
+	@OneToMany(mappedBy="job")
+	private Set<Interview> interviews = new HashSet<>();

```

Comment:
Perhaps add a short comment here, so it is similar to the rest of the file.

Reply:
Changes are made as per suggestion